config/printer-creality-ender3-s1plus-2022.cfg

```
@@ -129,3 +129,9 @@ runout_gcode: PAUSE
 
 [pause_resume]
 recover_velocity: 25
+
+[bed_screws]
+screw1: 27, 27 # Front Left
```

```suggestion
screw1: 27, 27 # Front Left
screw2: 267, 27 # Front Right
screw3: 267, 267 # Back Right
screw4: 27, 267 # Back Left
```
Remove the comments and this will be fine to merge imo since the normal sized S1's now have the same thing, but follow what James said and add a signed-off-by line